pkg/domain/infra/abi/images.go

```diff
@@ -226,7 +226,7 @@ func (ir *ImageEngine) Pull(ctx context.Context, rawImage string, options entiti
 	if err != nil {
 		imageRef, err = alltransports.ParseImageName(fmt.Sprintf("%s%s", dockerPrefix, rawImage))
 		if err != nil {
-			return nil, errors.Errorf("invalid image reference %q", rawImage)
```

Comment:
Does this produce really verbose errors?

If they get too gnarly, we may want to make this only happen when `--log-level=debug` is passed, and we return the previous output otherwise.

Reply:
Not particularly, here are some examples:

`Invalid image name "%s", expected colon-separated transport:reference`, imgName
`Invalid image name "%s", unknown transport "%s"`, imgName, parts[0]

Reply:
Alright. I think we should include Qi's suggestion about wrapping, but once that's done LGTM